main.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'app'
+
+def main
+  App.run
+end
+
+main
```

Comment:
- [ ] Base on the project requirement, you should make sure that your app behaves in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) but currently  if  execute your `main.rb` to run your project, I get this error saying you have an undefined method `run` for your App class so make sure fix this issue and ensure that your app work in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) :relaxed: 

  ![image](https://github.com/supreeti/OPPS-Library-Management-System/assets/92303891/76d3af4f-3ac1-4f11-90cf-680f128ef85f)

- [ ] As I mention before, this the `main.rb` file should serve as the entry point for your application, presenting users with a list of options to choose from. Users can then select an option, provide any necessary parameters, and exit the app like the example code below.

    ```ruby
    # main.rb
    def main
      app = App.new()
    
      response = nil
    
      # [OPTIONAL] You can learn  about the ruby lessons about basic sintax like loops: https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/lessons/basic_syntax.md  
      # *Present the user with a list of options to perform.
      puts "Welcome to School Library App!\n\n"
      while response != "7"
        puts "Please choose an option by enterin a number:"
        puts "1 - List all books"
        puts "2 - List all people"
        puts "3 - Create a person"
        puts "4 - Create a book"
        puts "5 - Create a rental"
        puts "6 - List all rentals for a given person id"
        puts "7 - Exit"
        # *Lets users choose an option.
        response = gets.chomp
    
        case response
        when "1"
          app.list_books()
        when "2"
          app.list_people()
        when "3"
          app.create_person()
        when "4"
          app.create_book()
        when "5"
          app.create_rental()
        when "6"
          app.list_rentals_for_person_id()
        when "7"
          # *Have a way to quit the app.
          puts "Thank you for using this app!"
        end
    
        puts "\n"
      end
    end
    
    # 'main' method should be invoked at the end of the main.rb file, hint in this lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point
    main()
    ```